lib/msal-browser/src/interaction_handler/PopupHandler.ts

```diff
@@ -126,8 +126,8 @@ export class PopupHandler extends InteractionHandler {
              */
             const width = window.innerWidth || document.documentElement.clientWidth || document.body.clientWidth;
             const height = window.innerHeight || document.documentElement.clientHeight || document.body.clientHeight;
-            const left = ((width / 2) - (popUpWidth / 2)) + winLeft;
-            const top = ((height / 2) - (popUpHeight / 2)) + winTop;
+            const left = ((width / 2) - (popUpWidth / 2)) + Math.max(0, winLeft);
+            const top = ((height / 2) - (popUpHeight / 2)) + Math.max(0, winTop);
```

Comment:
Hmm, what if `height` is 100 and popupHeight is 150? This would still result in a negative number, right? I think you need to wrap the entire calculate in `Math.max`:

`const top = Math.max(0, ((height / 2) - (popUpHeight / 2)) + winTop)`

Reply:
Right, let me update this. Sadly I can't repro the issue in either Windows 10 or macOS Catalina.